lib/route.js

```diff
@@ -320,7 +320,7 @@ function buildRouting (options) {
 
     var queryPrefix = req.url.indexOf('?')
     var query = querystringParser(queryPrefix > -1 ? req.url.slice(queryPrefix + 1) : '')
-    var request = new context.Request(id, params, req, query, childLogger)
+    var request = new context.Request(id, params, req, query, childLogger, context)
```

Comment:
@Eomm Is it the only place where we instantiate Request in runtime, or I missed some?